feature-libs/cart/base/root/facade/multi-cart.facade.ts

```diff
@@ -209,6 +210,16 @@ export abstract class MultiCartFacade {
    */
   abstract assignEmail(cartId: string, userId: string, email: string): void;
 
+  /**
+   * Remove cart
+   *
+   * Used for cleaning cart in local state, when we get information that it no longer exists in the backend.
+   * For removing particular cart in both places use DeleteCart actions.
```

Comment:
I'm confused by this comment. Which both places? Back-end and state?

Reply:
If so, maybe we can rephrase it somehow. ```To remove a cart on the back-end, please use `DeleteCart` actions.```
Moreover, why actions? We don't have a facade in front of those ngrx actions?

Reply:
Wei replied on Slack: https://sap-cx.slack.com/archives/C02L8BUATM5/p1654801507446859